sdk/eventhub/Microsoft.Azure.WebJobs.Extensions.EventHubs/tests/EventHubEndToEndTests.cs

```diff
@@ -712,14 +706,18 @@ public class EventHubTestInitialOffsetFromEnqueuedTimeJobs
 
             public static void ProcessMultipleEvents([EventHubTrigger(TestHubName, Connection = TestHubName)] EventData[] events)
             {
-                Assert.LessOrEqual(events.Length, ExpectedEventsCount);
+                Assert.GreaterOrEqual(events.Length, ExpectedEventsCount);
+
+                // there's potentially some level of rewind due to clock differences; allow a small delta when validating.
+                var earliestAllowedOffset = _initialOffsetEnqueuedTimeUTC.AddMilliseconds(-500);
```

Comment:
Where does this variable get used?

Reply:
It doesn't, but should have been used for the assertion on 725.   Fixed!